src/test/java/no/nav/fo/veilarbregistrering/registrering/bruker/db/BrukerRegistreringRepositoryDbIntegrationTest.java

```diff
@@ -127,57 +139,78 @@ public void skal_lagre_og_hente_registreringTilstand() {
         assertThat(lagretTilstand.getSistEndret()).isNull();
         assertThat(lagretTilstand.getStatus()).isEqualTo(Status.MOTTATT);
     }
 
     @Test
     public void skal_finne_den_eldste_tilstanden_med_status_mottatt() {
+        OrdinaerBrukerRegistrering registrering1 = gyldigBrukerRegistrering().setBesvarelse(BesvarelseTestdataBuilder.gyldigBesvarelse()
+                .setAndreForhold(AndreForholdSvar.JA));
+        OrdinaerBrukerRegistrering registrering2 = gyldigBrukerRegistrering().setBesvarelse(BesvarelseTestdataBuilder.gyldigBesvarelse()
+                .setAndreForhold(AndreForholdSvar.JA));
+        OrdinaerBrukerRegistrering registrering3 = gyldigBrukerRegistrering().setBesvarelse(BesvarelseTestdataBuilder.gyldigBesvarelse()
+                .setAndreForhold(AndreForholdSvar.JA));
+        OrdinaerBrukerRegistrering lagretRegistrering1 = brukerRegistreringRepository.lagre(registrering1, BRUKER);
+        OrdinaerBrukerRegistrering lagretRegistrering2 = brukerRegistreringRepository.lagre(registrering2, BRUKER);
+        OrdinaerBrukerRegistrering lagretRegistrering3 = brukerRegistreringRepository.lagre(registrering3, BRUKER);
+
+
         RegistreringTilstand eldsteRegistrering = registreringTilstand()
-                .brukerRegistreringId(112233L)
+                .brukerRegistreringId(lagretRegistrering1.getId())
                 .opprettet(LocalDateTime.now().minusMinutes(5))
                 .build();
         long id1 = brukerRegistreringRepository.lagre(eldsteRegistrering);
 
         RegistreringTilstand nyesteRegistering = registreringTilstand()
-                .brukerRegistreringId(332211)
+                .brukerRegistreringId(lagretRegistrering2.getId())
                 .opprettet(LocalDateTime.now().minusMinutes(1))
                 .build();
         brukerRegistreringRepository.lagre(nyesteRegistering);
 
         RegistreringTilstand midtersteRegistering = registreringTilstand()
-                .brukerRegistreringId(666666)
+                .brukerRegistreringId(lagretRegistrering3.getId())
                 .opprettet(LocalDateTime.now().minusMinutes(3))
                 .build();
         brukerRegistreringRepository.lagre(midtersteRegistering);
 
         Optional<RegistreringTilstand> lagretTilstand = brukerRegistreringRepository.finnNesteRegistreringForOverforing();
 
         assertThat(lagretTilstand.get().getId()).isEqualTo(id1);
-        assertThat(lagretTilstand.get().getBrukerRegistreringId()).isEqualTo(112233L);
+        assertThat(lagretTilstand.get().getBrukerRegistreringId()).isEqualTo(lagretRegistrering1.getId());
         assertThat(lagretTilstand.get().getStatus()).isEqualTo(Status.MOTTATT);
     }
 
     @Test
     public void skal_returnere_empty_naar_ingen_flere_mottatte() {
+        OrdinaerBrukerRegistrering registrering1 = gyldigBrukerRegistrering().setBesvarelse(BesvarelseTestdataBuilder.gyldigBesvarelse()
```

Comment:
Kunne vurdert å lage en kopi av gyldigBrukerRegistrering, men som samtidig satte besvarelsen så du slipper å gjøre det X antall ganger her. Det er ikke viktig for testen sin del.

F.eks. `brukerRegistreringMedBesvarelse()`.

Reply:
Innså at den endringen defaultbesvarelsen som setter ikke er nødvendig for testen i det hele tatt. Så det ble ganske mye penere nå